pkgs/development/python-modules/coloredlogs/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitHub, humanfriendly, verboselogs, pytest, pytestcov, capturer }:
+
+buildPythonPackage rec {
+  pname = "coloredlogs";
+  version = "10.0";
+
+  src = fetchFromGitHub {
+    owner  = "xolox";
+    repo   = "python-coloredlogs";
+    rev    = version;
+    sha256 = "0rdvp4dfvzhx7z7s2jdl3fv7x1hazgpy5gc7bcf05bnbv2iia54a";
+  };
+
+  checkPhase = ''
+    export PATH=$PATH:$out/bin
```

this overwrites all of the checkPhase
```suggestion
    PATH=$PATH:$out/bin pytest .
```